server/chat-plugins/mafia-data.js

```diff
@@ -530,12 +542,24 @@ const roles = {
 		id: `governor`,
 		memo: [`Governor: At the end of each day, the Governor can decide whether the person who would be lynched will die or not.`],
 	},
-	grey_goo: `gray_goo`,
-	gray_goo: {
-		name: `Gray Goo`,
-		id: `graygoo`,
+	gray_goo: `grey_goo`,
+	grey_goo: {
+		name: `Grey Goo`,
+		id: `greygoo`,
+		memo: [`You can target a player to turn them into more Grey Goo. Their alignment doesn't change.`],
+	},
+	graygooci: `greygoocrackingidea`,
+	gray_goo_ci: `greygoocrackingidea`,
+	gray_goo_cracking_idea: `greygoocrackingidea`,
+	graygoocrackingidea: `greygoocrackingidea`,
+	greygooci: `greygoocrackingidea`,
+	grey_goo_ci: `greygoocrackingidea`,
+	grey_goo_cracking_idea: `greygoocrackingidea`,
+	greygoocrackingidea: {
+		name: `Grey Goo (Cracking Idea)`,
+		id: `greygooci`,
```

Comment:
ID should not match an alias, but rather the main property eg: `greygoocrackingidea`